<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyPointsRegistrationControls</class>
 <widget class="QWidget" name="MyPointsRegistrationControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>634</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelWarning">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
     </property>
     <property name="text">
      <string>Please create two sets of points with 4 points each!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonPointSet1">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="text">
      <string>Create Point Set 1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonPointSet2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Create PointSet 2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="registrationButton">
     <property name="text">
      <string>Registration between Sets</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Transformation Matrix:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>RMSE (mm): </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
